docs/source/notebooks/Density_matrix_Fock_space.ipynb

```diff
@@ -428,7 +428,7 @@
     "collapsed": false
    },
    "source": [
-    "### Computating the expectation value of an operator"
+    "### Computation the expectation value of an operator"
```

Comment:
"Computing the ..." or "Computation of the ..."

Reply:
done https://github.com/Quandela/Perceval/pull/435/commits/c6b7ae8a1d88f689c4363901a18d24a76acb5897